### ISSUE TITLE
[Celia/Phil] Option to select 'coaches' or 'carriages'

### ISSUE THREAD

**Issue by opott** [davwheat/rail-announcements#232]
It would be good to have the option in both the announcement generator and in Live Trains to select if you want it to announce as 'coaches' or 'carriages'.

**Comment by Sebrh11** [comment]
There's no file for carriages in the ones we have, its only coaches.

**Comment by davwheat** [comment]
Correct, but I did rip out my own "carriages" for Celia from another audio file:

https://github.com/Rail-Announcements/ketech-llpa-announcements/blob/main/custom/Female1/e/carriages.mp3
